src/main/java/frc/robot/RobotContainer.java

```diff
@@ -309,18 +308,23 @@ private Command shootAuton() {
   }
 
   private Command ampAuton() {
-    return null; // return Commands.parallel(
-    //     trapElvCommandFactory.positionAMP(),
-    //     Commands.waitUntil(trapElvSubsystem.isAMPReady())
-    //         .andThen(trapElvCommandFactory.scoreAMP()));
+    return Commands.parallel(
+        trapElvCommandFactory.positionAMP(),
+        Commands.waitUntil(trapElvSubsystem.isAMPReady())
+            .andThen(trapElvCommandFactory.scoreAMP())
+            .onlyWhile(trapElvCommandFactory.getSourceBreak()));
   }
 
   // Register commands for auton
   public void registerCommands() {
     HashMap<String, Command> autonCommands = new HashMap<String, Command>();
 
-    autonCommands.put("Shoot", shootAuton());
-    // autonCommands.put("Amp", ampAuton());
+    if (enabledSubsystems.shooterEnabled) {
```

Comment:
Shouldn't need this if we setup our command factories correctly.
Same with other subsystem enabled checks in this method